docs/reference/bpmn-processes/multi-instance/multi-instance.md

```diff
@@ -105,6 +115,9 @@ A sequential multi-instance service task:
           inputCollection="= items" inputElement="item"
           outputCollection="results" outputElement="= result" />
     </bpmn:extensionElements>
+    <bpmn:completionCondition xsi:type="bpmn:tFormalExpression">
+        = true
```

Comment:
❌  Let's improve this example. 

We could use the `result` or `results` variables in an expression, e.g. `= result.isSuccessful` or `= count(results[item = true]) > 3` (at least 3 times the value `true` in the results list). We could also just use the moment of completion, something like: `= list contains([6,7], today().weekday)` to check that it's the weekend.